scripts/circle_deploy.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+NPM_PACKAGE_NAME=$(json -f package.json name)
+NPM_PACKAGE_VERSION=$(json -f package.json version)
+
+#
+# configure git
+#
+git config --global user.name "tadeploy"
+git config --global user.email "devteam@technologyadvice.com"
+
+#
+# build
+#
+gulp docs
+
+#
+# generate changelog
+#
+echo "...generating changelog"
+github_changelog_generator $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
+
+#
+# push to master and gh-pages
+#
+git add .
+
+if [[ -n $(git status --porcelain) ]]; then
+  echo "...starting push, repo is dirty after build"
+  git commit -n -m "deploy commit by $CIRCLE_USERNAME [ci skip]"
+  git push origin master
+  git push -f origin master:gh-pages
+else
+  echo "...skipping push, repo is clean after build"
+fi
+
+#
+# s3 sync
+#
+SOURCE_DIRECTORY=./dist
+S3_BUCKET=ta-stardust-assets
+
+echo "...syncing with s3"
+aws s3 sync ${SOURCE_DIRECTORY} s3://${S3_BUCKET}/${NPM_PACKAGE_VERSION}/ --delete --acl public-read
```

Comment:
Right now, Stardust does not have a `./dist`.  We plan to, once we do, we will already have CDN distributions working.